app/main.py

```diff
@@ -2,0 +1,14 @@
+
+def copy_file(command: str) -> None:
+    command_parts = command.split()
```

Comment:
Code Style: The variable name `command_parts` is not descriptive enough. Consider renaming it to something more meaningful, such as `command_elements` or `command_components`.